static_src/css/main.css

```diff
@@ -36,3 +36,4 @@ input[type="checkbox"] + label::before,
 input[type="radio"] + label::before {
   display: none;
 }
+
```

Comment:
Yea we were using it for quick styles at first, but found that putting it in cg-style was easier.